src/schema/causality_jwt.js

```diff
@@ -43,16 +48,15 @@ export default {
       // For logged in and...
     } else if (options.role === "PARTICIPANT" && accessToken) {
       return Promise.all([
-        gravity(`sale/${options.sale_id}`),
-        gravity.with(accessToken)("me"),
-        gravity.with(accessToken)("me/bidders", { sale_id: options.sale_id }),
+        saleLoader(options.sale_id),
+        meBiddersLoader({ sale_id: options.sale_id }),
       ]).then(([sale, me, bidders]) => {
```

Comment:
the order will have changed here from `sale, me, bidders` to `sale, bidders` - I'm pretty sure this will need updating

Reply:
ah yea whoops! good catch!!

Luckily there's a good amount of failing tests in this PR, and this resolver specifically is tested, so 👍